dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -1332,7 +1332,7 @@ public void printWesHelpFooter() {
         out("NOTE: WES SUPPORT IS IN BETA AT THIS TIME. RESULTS MAY BE UNPREDICTABLE.");
     }
 
-    private void publishHelp() {
+    protected void publishHelp() {
```

Comment:
If I'm reading this right, you are overriding this implementation in both/all subclasses, and don't call `super`, so this code will never get executed. If that's so, then you should make this, with no body.

```
abstract protected void publishHelp();
```